DayrSuryan/DSEthiop12.xml

```diff
@@ -143,6 +140,21 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         ሀብተ፡ ጸጋ፡ ወኃይል፡ ለከ፡ እንዘ፡ ንዌድስ፡ ወኢንዓርፍ፡ ለዝሉፉ፡ በአልባቢነ፡ ናስተማስል፡ አምሳለ፡ መንግሥትከ፡ ቦቱ፡ በእንቲአከ፡ ወበእንተ፡ ፍቁር፡ ወልድከ፡ እንግዚእነ፡ ኢየሱስ፡ ዘቦቱ፡ ለከ፡ ስብሐት፨
                         ወእኂዝ፡ ለዓለመ፡ ዓለም፡ አሚን። ። ። ። ።
                      </explicit>
+                     <note>The incipit (<foreign xml:lang="gez">ዘነገሮሙ፡ እግዚእነ፡</foreign>) is similar to that in some manuscript
```

Comment:
I am not sure, do you think this can be somehow meaningfully summarised? I can also look into it with more concentration later.

Reply:
this is meaningful, in a classical printed catalogue this would be a footnote.

Reply:
I can put it back into <-- -->, and leave only the summary statement. This would be pity, however.

Reply:
No, no, then better to keep it.